crash_similarity.py

```diff
@@ -68,18 +68,19 @@ def get_stack_trace_from_crashid(crash_id):
 
 
 def get_stack_traces_for_signature(fnames, signature):
```

Comment:
Can you add a parameter, `traces_num`, with a default of 100, and use it in the request with `_facets_size`?